contracts/examples/guards/DebugTransactionGuard.sol

```diff
@@ -96,23 +95,22 @@ contract DebugTransactionGuard is BaseGuard {
     }
 
     /**
-     * @notice Called by the Safe contract before a transaction is executed via a module.
-     * @param to Destination address of Safe transaction.
-     * @param value Ether value of Safe transaction.
-     * @param data Data payload of Safe transaction.
-     * @param operation Operation type of Safe transaction.
-     * @param module Account executing the transaction.
-     * @return moduleTxHash Hash of the module transaction.
+     * @notice Called by the Safe contract before a module transaction is executed.
+     * @param to The address to which the transaction is intended.
+     * @param value The value of the transaction in Wei.
+     * @param data The transaction data.
+     * @param operation The type of operation of the module transaction.
+     * @param msgSender The address of the message sender.
      */
     function checkModuleTransaction(
         address to,
         uint256 value,
         bytes memory data,
         Enum.Operation operation,
-        address module
+        address msgSender
```

Comment:
Why was the name changed from module to `msgSender`? The interface uses `module`

Reply:
ooh.. This was changed when I replaced `checkTransaction` with `checkModuleTransaction`. Will revert it.